client/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="index.css">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+		<title> 
+				Code Runner
+		</title> 
+		<link rel = "icon" href = "images/logo.png"  type = "image/x-icon">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+	</head>
+	<body>
+	<div class="container top-container">
+			 <div class="header" id="app-header">
+			 <div class="side-nav sn-bg-4 mdb-sidenav fixed">
+				<img src="images/logo.png" alt="logo" height="40" class="pull-left heading" />
+				<h1>Code Runner</h1>
+			</div>
+			</div>
+			<div class="content"> 
+			<form id="myForm">
+				<div class="form-group row">
+					<label for="assgn" class="col-sm-2 col-form-label">Assignment:</label>
+					<div class="col-sm-5">
+						<input type="file" id="file" name="file" class="form-control" onchange="verifyFiles()">
+					</div>
+					<div class="col-sm-5 file-error">
+						<span id="fileError" class="text-danger"></span>
+					</div>
+				</div>
+				<div class="form-group row">
+					<label for="Cmd" class="col-sm-2 col-form-label">Command line arguments:</label>
+					<input type="button" value="Add" id="addArg" onclick="appendRow()" class="btn">
+				</div>
+				<div id="cmdArgs">
+				</div>
+				<div class="form-group row">
+					<input type="submit" value="Submit" id="submitForm" class="btn btn-primary">
+					<input type="button" value="Build & Run" class="btn btn-info">
+				</div>
+				<div class="form-group row">
+					<label for="Output" class="col-sm-2 col-form-label">Output:</label><br>
+					<textarea rows="13" name="output" class="form-control" id="output" readonly="true"></textarea>
```

Comment:
why 13 rows?

Reply:
I just left it as 13 for now. But I was going to make the output area size dynamic later

Reply:
Yes. That is done then.